docs/topics/templates/document-attributes.adoc

```diff
@@ -39,6 +39,7 @@ ifdef::mta[]
 :ocp-short: OpenShift
 :ocp: OCP
 :ocp-version: 4.17
+:ocp-supported: 4.17, 4.16, 4.15
```

Comment:
@anarnold97 ,
MTA 7.2.1 releases on Feb 20, 2025,
and OCP-4.18 on Feb 25-Feb 28 2025.
QE tests were run on OCP-4.16, 4.17, 4.18 (mainly on 4.18). 
Therefore, once OCP-4.18 is released, I suggest to have this updated into:

:ocp-version: 4.18
:ocp-supported: 4.18, 4.17, 4.16